__tests__/nc-news.test.js

```diff
@@ -25,6 +25,31 @@ describe('GET /api/topics', () => {
   });
 });
 
+describe('GET /api/articles/:article_id', () => {
+  it('200: return article by given Id', async () => {
+    const {body} = await request(app).get('/api/articles/1').expect(200);
+
+    expect(body.article).toBeInstanceOf(Object);
+    expect(body.article).toMatchObject({
+      author: expect.any(String),
+      title: expect.any(String),
+      article_id: expect.any(Number),
```

Comment:
If we are testing for an article by the id e.g /api/articles/1 we can be more specific with the test. Instead of expecting any Number for the article_id we would expect back the Number 1 etc.